en/08_Changelogs/6.1.0.md

```diff
@@ -99,6 +100,27 @@ If you are using the [`DataObjectSchema::databaseIndexes()`](api:SilverStripe\OR
 
 See the [indexes documentation](/developer_guides/model/indexes/) for more details.
 
+### Generated column support {#generated-columns}
+
+Generated columns are database columns where the value is generated inside the database, rather than being set by a user. They're usually based on other columns in the database and can be either generated when the record is updated and stored, or generated when requested in which case they aren't stored.
+
+Some good use cases for generated columns include:
+
+- sorting in a gridfield on a complex summary field: It's common to use a getter method to get some value derived from your database fields (e.g. a discounted price), and use that in `summary_fields`. With a generated column, you can remove the getter method and the [`GridFieldSortableHeader`](api:SilverStripe\Forms\GridField\GridFieldSortableHeader) component will be able to sort using the column.
+- using functional indexes: generated columns can be included in indexes, allowing you to sort or filter by complex expressions in an efficient way.
+- data integrity: unlike generating values inside an `onBeforeWrite()` extension hook, generated column values will be correct even if you update the record with raw SQL.
+- reduce repetition: instead of using a complex expression in multiple different places, you can just give the expression a name with a generated column.
+
+See [data types and casting](/developer_guides/model/data_types_and_casting/#generated-columns) for details about using generated columns.
+
+#### Support in other SQL servers
+
+Generated column support has been added to the MySQL database connector in `silverstripe/framework`, but is not guaranteed to work for other database servers.
+
+If you maintain a module that adds support for another database server, you'll need to implement the new [`DBSchemaManager::makeGenerated()`](SilverStripe\ORM\Connect\DBSchemaManager::makeGenerated()) and [`DBSchemaManager::needRebuildColumn()`](SilverStripe\ORM\Connect\DBSchemaManager::needRebuildColumn()) methods.
```

Comment:
```suggestion
If you maintain a module that adds support for another database server, you'll need to implement the new [`DBSchemaManager::makeGenerated()`](SilverStripe\ORM\Connect\DBSchemaManager::makeGenerated()) and [`DBSchemaManager::needRebuildColumn()`](SilverStripe\ORM\Connect\DBSchemaManager::needRebuildColumn()) methods to support generated columns.
```

Presumbably you don't HAVE to update the the module, because there's no BC changes?

Reply:
You do HAVE to update the module to support generated columns if you want to support generated columns lol. I think "to support generated columns" is very heavily implied but I'll add it to remove any hint of confusion.